src/Uno.UI/UI/Xaml/Controls/Border/Border.Android.cs

```diff
@@ -102,7 +102,7 @@ protected override void OnBackgroundChanged(DependencyPropertyChangedEventArgs e
 
         partial void OnPaddingChangedPartial(Thickness oldValue, Thickness newValue)
         {
-            UpdateBorder(true);
+            UpdateBorder(oldValue != newValue);
```

Comment:
Why do you need this? In general the DependencyProperty callback will only be raised if the DP is really different.

Reply:
It was just to only redraw instead of requesting layout but indeed, the callback is raised only if the DP really changed.
I will remove this.